asset-store/README.md

```diff
@@ -0,0 +1,100 @@
+# Asset Store
+
+## Overview
+
+This example illustrates how to use [asset-store](https://kyma-project.io/docs/1.5/components/asset-store/) to store simple webpages.
+
+## Prerequisites
+
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- [Kyma](https://kyma-project.io/docs/) as the target deployment environment
+
+## Details
+
+By default minio store all resources on current cluster, but it works fine also on cloud storage systems. Read [asset-store tutorials](https://kyma-project.io/docs/components/asset-store#tutorials-tutorials) for more informations.
+
+### Installation
+
+1. Run the export GH_WEBPAGE_URL={value}, where value contain address to download webpage
+
+    Example:
+
+    ```bash
+    export GH_WEBPAGE_URL=https://github.com/pPrecel/simple-page-for-asset-store/archive/master.zip
+    ```
+
+2. Apply bucket CR:
+
+    ```
+    cat <<EOF | kubectl apply -f -
+    apiVersion: assetstore.kyma-project.io/v1alpha2
+    kind: Bucket
+    metadata:
+      name: pages
+      namespace: default
+    spec:
+      region: "us-east-1"
+      policy: readwrite
+    EOF
+    ```
+
+3. Apply asset CR:
+
+    ```
+    cat <<EOF | kubectl apply -f -
+    apiVersion: assetstore.kyma-project.io/v1alpha2
+    kind: Asset
+    metadata:
+      name: webpage
+      namespace: default
+    spec:
+      source:
+        url: ${GH_WEBPAGE_URL}
+        mode: package
+      bucketRef:
+        name: pages
+    EOF
+    ```
+
+### Testing
+
+1. Describe asset CR:
+
+    ```bash
+    kubectl describe assets.assetstore.kyma-project.io webpage
+    ```
+    
+2. Find "Asset Ref" field and merge "Base URL" with file name of your index.html
+
+    Example:
```

Comment:
Line 70 should be blank (we should leave one line of space before a code block).